### PR TITLE
Additional notes on releasing resources when using file streams

### PR DESCRIPTION
**Pull request by fwqaaq**
See: <https://github.com/denoland/deno/issues/23481>

### CODE REVIEW

File: deploy/api/runtime-fs.md [hunk 1]
```diff
@@ -229,6 +229,14 @@ async function handler(_req) {
 Deno.serve(handler);
 ```
 
+> Note: when you iterate over a file stream as shown below, the file descriptor will be automatically closed at the end of iteration. So, there is no need to manually close the file descriptor.
+>
+> See: <https://github.com/denoland/deno/issues/23481>
```
Comment (fwqaaq) [inline deploy/api/runtime-fs.md:234]:
```suggestion
> Note: when you iterate over a file stream as shown below, the file descriptor
> will be automatically closed at the end of iteration. So, there is no need to
> manually close the file descriptor.
>
> See: <https://github.com/denoland/deno/issues/23481>
```

File: deploy/api/runtime-fs.md [hunk 2]
```diff
@@ -229,6 +229,16 @@ async function handler(_req) {
...
+> Note: when you iterate over a file stream as shown below, the file descriptor
+> will be automatically closed at the end of iteration. So, there is no need to
+> manually close the file descriptor.
+>
+> See: <https://github.com/denoland/deno/issues/23481>
+
+```ts
+const iterator = fd.readable[Symbol.asyncIterator]();
+```
+
```
Comment (thisisjofrank) [inline deploy/api/runtime-fs.md:241]:
```suggestion
:::note

When you iterate over a file stream as shown below, the file descriptor
will be automatically closed at the end of iteration. There is no need to
manually close the file descriptor.

```ts
const iterator = fd.readable[Symbol.asyncIterator]();
```

:::
```

File: deploy/api/runtime-fs.md [hunk 3]
```diff
@@ -229,6 +229,14 @@ async function handler(_req) {
...
 ```
 
+:::note
+
+When you iterate over a file stream as shown below, the file descriptor
+will be automatically closed at the end of iteration. There is no need to
+manually close the file descriptor.
+
+```ts
+const iterator = fd.readable[Symbol.asyncIterator]();
```
Comment (thisisjofrank) [inline deploy/api/runtime-fs.md:239]:
```suggestion
:::note

When you iterate over a file stream as shown below, the file descriptor
will be automatically closed at the end of iteration. There is no need to
manually close the file descriptor.

```ts
const iterator = fd.readable[Symbol.asyncIterator]();
```

:::

Comment (thisisjofrank) [inline deploy/api/runtime-fs.md:239]:
```suggestion
:::note

When you iterate over a file stream as shown below, the file descriptor
will be automatically closed at the end of iteration. There is no need to
manually close the file descriptor.

```ts
const iterator = fd.readable[Symbol.asyncIterator]();
```
:::

File: deploy/api/runtime-fs.md [hunk 4]
```diff
@@ -229,6 +229,14 @@ async function handler(_req) {
...
 ## Deno.File
```
Comment (thisisjofrank) [inline deploy/api/runtime-fs.md:240]:
```suggestion

:::

## Deno.File
```

File: deploy/api/runtime-fs.md [hunk 5]
```diff
@@ -229,6 +229,17 @@ async function handler(_req) {
...
 
+:::note
+
+When you iterate over a file stream as shown below, the file descriptor
+will be automatically closed at the end of iteration. There is no need to
+manually close the file descriptor.
+
+```ts
+const iterator = fd.readable[Symbol.asyncIterator]();
+
```
Comment (thisisjofrank) [inline deploy/api/runtime-fs.md:240]:
```suggestion
```
```

Comment (thisisjofrank) [inline deploy/api/runtime-fs.md:240]:
```suggestion

`const iterator = fd.readable[Symbol.asyncIterator]();`

```

**Review by thisisjofrank — APPROVED** [review]

**Pull request merged**